src/ORM/FieldType/DBFloat.php

```diff
@@ -35,34 +35,34 @@ public function requireField()
      *
      * @uses number_format()
      */
-    public function Nice()
+    public function Nice(): string
     {
         return number_format($this->value ?? 0.0, 2);
     }
 
-    public function Round($precision = 3)
+    public function Round($precision = 3): float
     {
         return round($this->value ?? 0.0, $precision ?? 0);
     }
 
-    public function NiceRound($precision = 3)
+    public function NiceRound($precision = 3): string
     {
         return number_format(round($this->value ?? 0.0, $precision ?? 0), $precision ?? 0);
     }
 
-    public function scaffoldFormField($title = null, $params = null)
+    public function scaffoldFormField(?string $title = null, array $params = []): ?FormField
     {
         $field = NumericField::create($this->name, $title);
         $field->setScale(null); // remove no-decimal restriction
         return $field;
     }
 
-    public function nullValue()
+    public function nullValue(): ?int
```

Comment:
Arguably should be `float|int|null` but realistically `0.0` and `0` are identical, and `0.0` is the only float that would make sense here.